face_alignment/api.py

```diff
@@ -27,7 +27,7 @@ class NetworkSize(IntEnum):
 
 
 class FaceAlignment:
-    def __init__(self, landmarks_type, face_align_model_path, depth_pred_model_path, network_size=NetworkSize.LARGE,
+    def __init__(self, landmarks_type, face_align_model_path, depth_pred_model_path=None, network_size=NetworkSize.LARGE,

```

Comment:
Does this work when `depth_pred_model_path` is None?

Reply:
Yes, by using this 'None' variable, we can keep the structure in case we need it in the future. Now it is not used!

Reply:
Got it ✌️